pkg/version/postgres/gen.go

```diff
@@ -1,3 +1,3 @@
 package postgres
 
-//go:generate pg-table-bindings-wrapper --type=storage.Version --singleton
```

Comment:
Why do we need this change? If you added this then please remove `// Code generated by pg-bindings generator. DO NOT EDIT.` from corresponding `store.go`

Reply:
@janisz I needed to move the simple version store to pkg so that both central and migrator could get at it.  Once I did that that I either had to remove some of the SAC stuff such as resources from the store or pull all that stuff to pkg as well.  Due to what this store is, we decided in the short term to just generate the schema, manually remove the SAC stuff causing issue, and re-visit in [ROX-12083](https://issues.redhat.com/browse/ROX-12083) if we desire.  We missed removing the comment from the store.  I will take care of that as I'm working other PRs that are doing follow on things in the feature.  Hopefully that answers your questions.

Reply:
OK, sounds good. Pleas add todos so it will be obvious to everybody that there is something that will be changed